src/endpoints/deactivateDevices.ts

```diff
@@ -0,0 +1,26 @@
+import { requestUserApi } from '../requestApi';
+import { Secrets } from '../types';
+
+interface DeactivateDevicesParams {
```

Comment:
i recommend taking the types from node _d.ts files that will also contain the description 

in the future we will make the generation automatic

Reply:
WDYM ? Which _d.ts files ?